components/Footer/Footer.tsx

```diff
@@ -0,0 +1,58 @@
+import clsx from "clsx";
+import { useTranslation } from "react-i18next";
+import Image from '@/components/Image'
+import Link from "next/link";
+
+const flexItemsCenter: string = clsx('flex items-center')
+
+const Footer = () => {
+  const { t } = useTranslation('common');
+
+  return (<footer className={clsx('w-full mx-auto')}>
+    <div className=" bg-gray-50">
+      <div
+        className={`layout ${flexItemsCenter} justify-end py-3`}
+      >
+        <span className={clsx('mx-4')}>
+          {t('incubated.by')}
+        </span>
+
+        <a href="https://code4.ro/ro/code-for-romania-war-task-force">
+          <Image src={'/code_logo_colored.svg'} alt="Code 4 Romania logo" className="h-[42px]" />
+        </a>
+      </div>
+    </div>
+    <footer className="text-white [background-color:#00478F]">
```

Comment:
can we add this color to the `tailwind.config.js` file . I think it's the darkest blue we use so add it in the blue array as `950: "#00478F"`

Reply:
done